bundles/event_admin/remote_provider/remote_provider_mqtt/src/celix_earpm_impl.c

```diff
@@ -148,6 +148,7 @@ celix_event_admin_remote_provider_mqtt_t* celix_earpm_create(celix_bundle_contex
     }
 
     if (asprintf(&earpm->syncEventAckTopic, CELIX_EARPM_SYNC_EVENT_ACK_TOPIC_PREFIX"%s", earpm->fwUUID) < 0) {
+        earpm->syncEventAckTopic = NULL;
```

Comment:
This assignment operation may be meaningless, because if an error is made here, `earpm` and `earpm->asyncEventAckTopic` will not be used.